src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.js

```diff
@@ -0,0 +1,268 @@
+
+    processedQuery: function() {
+      return this.query.trim().toLowerCase()
+    },
+    activePlaylists: function() {
+      // Very rare that a playlist name only has 1 char
+      if (this.processedQuery.length === 0) { return this.allPlaylists }
+
+      return this.allPlaylists.filter((playlist) => {
+        if (typeof (playlist.playlistName) !== 'string') { return false }
+
+        return playlist.playlistName.toLowerCase().includes(this.processedQuery)
+      }).sort((a, b) => {
+        // Latest updated first
+        return b.lastUpdatedAt - a.lastUpdatedAt
+      })
+    },
+
+    createPlaylistPromptShown: function () {
+      return this.$store.getters.getShowCreatePlaylistPrompt
+    },
+
+    tabindex() {
+      return this.interactionsLocked ? -1 : 0
+    },
+
+    sortBySelectNames() {
+      return Object.values(SORT_BY_VALUES).map((k) => {
+        switch (k) {
+          case SORT_BY_VALUES.NameAscending:
+            return this.$t('User Playlists.Sort By.NameAscending')
+          case SORT_BY_VALUES.NameDescending:
+            return this.$t('User Playlists.Sort By.NameDescending')
+          case SORT_BY_VALUES.LatestCreatedFirst:
+            return this.$t('User Playlists.Sort By.LatestCreatedFirst')
+          case SORT_BY_VALUES.EarliestCreatedFirst:
+            return this.$t('User Playlists.Sort By.EarliestCreatedFirst')
+          case SORT_BY_VALUES.LatestUpdatedFirst:
+            return this.$t('User Playlists.Sort By.LatestUpdatedFirst')
+          case SORT_BY_VALUES.EarliestUpdatedFirst:
+            return this.$t('User Playlists.Sort By.EarliestUpdatedFirst')
+          default:
+            console.error(`Unknown sortBy: ${k}`)
+            return k
+        }
+      })
+    },
+    sortBySelectValues() {
+      return Object.values(SORT_BY_VALUES)
+    },
+  },
+  watch: {
+    allPlaylistsLength(val, oldVal) {
+      // When playlist length changed, a playlist removed or added
+      // Only cares about playlist added
+      if (val < oldVal) { return }
+
+      this.allPlaylists.forEach((playlist) => {
+        // Old playlists don't have `createdAt`
+        if (playlist.createdAt == null) { return }
+        // Only playlists created after this prompt shown should be considered
+        if (playlist.createdAt < this.lastShownAt) { return }
+        // Only playlists not auto added to selected yet should be considered
+        if (this.createdSincePromptShownPlaylistIdList.includes(playlist._id)) { return }
+
+        // Add newly created playlists to selected ONCE
+        this.createdSincePromptShownPlaylistIdList.push(playlist._id)
+        this.selectedPlaylistIdList.push(playlist._id)
+      })
+
+      // Focus back to search input
+      // Allow search and easier deselecting new created playlist
+      this.$refs.searchBar.focus()
+    },
+
+    createPlaylistPromptShown(value) {
+      // Lock interactions when create playlist prompt opened
+      this.interactionsLocked = value
```

Comment:
**question:** For what purpose are we locking the search bar? Could you document it in the code?

Reply:
Also, it seems to be just making it non-keyboard-focusable, not disabling it. If you want to disable it, you can use the `disabled` prop on `ft-input`.

Reply:
**suggestion (blocking):** If you do end up needing to keep this in as is, please replace the `inputTabindex` `number` prop with a `boolean` `isInputKeyboardNavigable`. That way it's less declarative and has no need of handling odd inputs.

Reply:
Code removed due to the issue I experienced at early stage of dev seems no longer reproducible

Reply:
Thanks! Can you also remove the `input-tabindex` prop from `ft-input`s?

Reply:
Just did